packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -841,6 +844,27 @@ export default class LocusInfo extends EventsScope {
     }
   }
 
+  /**
+   * @param {Object} embeddedApps
+   * @returns {undefined}
+   * @memberof LocusInfo
+   */
+  updateEmbeddedApps(embeddedApps) {
```

Comment:
i'm a little concerned about performance here especially in locus events. locus has *lots* of events and `isEqual` does deep comparison of everything in `embeddedApps`. so if `embeddedApps` has quite large, nested objects, i think this could get to be a little bit of a bottle neck. is there some other way we do this `!isEqual(this.embeddedApps, embeddedApps)` better? what does each object in `embeddedApp` look like? is it possible to use some other property on them?

Reply:
Good point.  I think I can test the length of the embeddedApps array and the state of the first app to determine if it has changed.

Reply:
doublechecking...
so they are always in the same order?
and we only care about insert/deletes? we don't need to know if something changed within them?